testdriver/testplan.py

```diff
@@ -523,15 +523,17 @@ def send_one_line(self, input_line):
                 # TODO!!!! Return an error for the offending line instead of failing for the whole batch
 
                 return None
-                input = json.loads(input_line.replace('#EXIT', ''))
-                error_result = {'label': input['label'],
-                                'input_data': input,
-                                'error': self.run_error_message
-                }
+                try:
```

Comment:
insert an empty to visually separate the try/except block

Reply:
also: do you even need the try/except block here? see if you can remove it

Reply:
Changed this. PTAL.